index.html

```diff
@@ -793,15 +786,13 @@ <h2><dfn>readable</dfn> attribute</h2>
         </ol>
 
       <li>
-        [=ReadableStream/Set up=] |stream| with
-        <a href="https://streams.spec.whatwg.org/#readablestream-set-up-pullalgorithm">pullAlgorithm</a>
+        [=ReadableStream/Set up with byte reading support=] |stream| with
+        <i><a href="https://streams.spec.whatwg.org/#readablestream-set-up-with-byte-reading-support-pullalgorithm">pullAlgorithm</a></i>
         set to |pullAlgorithm|,
-        <a href="https://streams.spec.whatwg.org/#readablestream-set-up-cancelalgorithm">cancelAlgorithm</a>
-        set to |cancelAlgorithm|,
-        <a href="https://streams.spec.whatwg.org/#readablestream-set-up-highwatermark">highWaterMark</a>
-        set to [=this=].{{[[bufferSize]]}}, and
-        <a href="https://streams.spec.whatwg.org/#readablestream-set-up-sizealgorithm">sizeAlgorithm</a>
-        set to a byte-counting size algorithm.
+        <i><a href="https://streams.spec.whatwg.org/#readablestream-set-up-with-byte-reading-support-cancelalgorithm">cancelAlgorithm</a></i>
+        set to |cancelAlgorithm|, and
+        <i><a href="https://streams.spec.whatwg.org/#readablestream-set-up-with-byte-reading-support-highwatermark">highWaterMark</a></i>
```

Comment:
Same with this one... not in the spec?

Reply:
None of these are in the spec yet; this PR depends on https://github.com/whatwg/streams/pull/1130.

Reply:
Ah, ok. Hopefully they should be all good once that's merged. Any linking/respec issues, just ping 😇